src/ngx_lua_resty_lmdb_module.c

```diff
@@ -139,13 +180,41 @@ static ngx_int_t ngx_lua_resty_lmdb_init_worker(ngx_cycle_t *cycle)
         return NGX_ERROR;
     }
 
+    if (lcf->key_file.data) {
+
+        u_char *key = lcf->key_file.data;
+
+        MDB_val enckey;
+        enckey.mv_data = &key[0];
+        enckey.mv_size = 32;
+
+        cipher = (EVP_CIPHER *)EVP_aes_256_gcm();
```

Comment:
this will needs to be determined by a directive, user can choose between `aes-256-gcm` and `chacha20-poly1305`